flashfocus/xutil.py

```diff
@@ -74,7 +74,7 @@ def set_opacity(window, opacity, checked=True):
     If opacity is None, request is ignored.
 
     """
-    if opacity:
+    if opacity >= 0:
```

Comment:
Nice catch! I think this fix won't work though - `if opacity is not None` should catch it.